paper-badge.html

```diff
@@ -153,11 +153,21 @@
         icon: {
           type: String,
           value: ''
+        },
+
+        /**
+         * Treated as the hidden attribute.
+         */
+        hidden: {
```

Comment:
Problem was with the test doing Polymer.Base.async instead of requestanimationframe. No longer necessary